src/forwarder.ts

```diff
@@ -51,7 +51,7 @@ export async function addCloudWatchForwarderSubscriptions(service: Service, aws:
 
     const canSub = await canSubscribeLogGroup(aws, logGroupName, expectedSubName);
     if (!canSub) {
-      errors.push(`Subscription already exists for log group ${logGroupName}. Skipping subscribing Datadog forwarder.`);
+      errors.push(`Two non Datadog Subscriptions already exist for the log group ${logGroupName}. Skipping subscribing from Datadog forwarder.`);
```

Comment:
Technically, we don't know for sure that there were exactly two non-Datadog subscriptions. While that would be the case for now, in the future AWS could add support for a third subscription (or more). Therefore, I would rephrase this to `Could not subscribe Datadog Forwarder due to too many existing subscription filter(s) for ${logGroupName}.`